se_mailbox/filelock.py

```diff
@@ -34,16 +34,6 @@ def __init__(self, file_name, timeout=10, delay=0.05, base=None,
         self.timeout = timeout
         self.delay = delay
         self.fd = None
-        if max_age and os.path.exists(self.lockfile):
```

Comment:
Why is this removed?

Reply:
@RoxanaButuc unanswered query here.

Reply:
Changes were made based on this [comment](https://jira.solarwinds.com/browse/MMA-868?focusedCommentId=1714710&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1714710) - in order to get the package up to date with mail-core